internal/net/http/routing/router.go

```diff
@@ -51,7 +51,7 @@ func New(opts ...Option) http.Handler {
 
 		rt.Handle(route.Pattern,
 			r.routing(route.Name, route.Pattern,
-				route.Methods, route.HandlerFunc))
+				route.Methods, route.HandlerFunc)).Name(route.Name)
```

Comment:
why do we need name??

Reply:
@kpango the reason is written here. 
https://github.com/vdaas/vald/pull/1214#issuecomment-836116290

The main reason is to able to get the route reference inside mux in test case